modern/tests/test_installed/test_conan.py

```diff
@@ -3,8 +3,10 @@ def test_conan_version(container, expected):
     assert out.strip() == f'Conan version {expected.conan}', f"out: '{out}' err: '{err}'"
 
 def test_revisions_enabled(container):
-    out, err = container.exec(['conan', 'config', 'get', 'general.revisions_enabled'])
-    assert out.strip() == '1', f"out: '{out}' err: '{err}'"
+    out, _ = container.exec(['conan', '--version'])
+    if "Conan version 1." in out.strip() and "base-" not in container.image:
```

Comment:
```suggestion
    if "Conan version 1." in out.strip():
```

Use instead `pytest.mark.service('deploy', 'jenkins')`. It will run this test only for those images